packages/inertia/src/inertia.js

```diff
@@ -63,6 +64,13 @@ export default {
       region.scrollTop = 0
       region.scrollLeft = 0
     })
+
+    if (window.location.hash) {
+      const el = document.getElementById(window.location.hash.slice(1))
+      if (el) {
+        el.scrollIntoView()
+      }
```

Comment:
You should be able to remove the condition thanks to the [optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) :+1: 

```suggestion
      document.getElementById(window.location.hash.slice(1))?.scrollIntoView()
```

Reply:
Are you sure this works on method calls? I'm getting errors when I try.

Reply:
Hmm, yes, definitely works. I've used it in the past. You can try on this page too: 

```js
// Will scroll you to the comment box
document.getElementById('issue-comment-box')?.scrollIntoView()


// Won't scroll and won't throw
document.getElementById('dummy-id-blah')?.scrollIntoView()
```

What error are you getting? If it's trouble to debug, this is not important to change anyway. Just a nice syntactic sugar tip.

Reply:
So I think it's just eslint complaining. It does compile. 👍

Reply:
This is going to require an eslint upgrade, which I'll do later. Going to leave it like this for now.

Reply:
Or refactor to TypeScript 😏